include/drivers/sensor.h

```diff
@@ -174,6 +174,16 @@ enum sensor_channel {
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
 
+	SENSOR_CHAN_RTC_CONTROL_BITS,
+
+	SENSOR_CHAN_RTC_TIME,
+
+	SENSOR_CHAN_RTC_ALARM_TIME,
+
+	SENSOR_CHAN_SQUARE_WAVE_FREQ,
+
+	SENSOR_CHAN_RESET_OSCILLATOR,
```

Comment:
Channels should be documented, since they don't look "standard" I'm not sure if they should be "private", @MaureenHelm ?